src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -35,40 +41,55 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             this.extensionHost = host;
             this.solutionTracker = activeSolutionTracker;
 
-            this.extensionHost.VisualStateManager.BindingStateChanged += this.OnBindingStateChanged;
-            this.solutionTracker.ActiveSolutionChanged += this.OnActiveSolutionChanged;
+            this.solutionBindingInformationProvider = this.extensionHost.GetService<ISolutionBindingInformationProvider>();
+            this.solutionBindingInformationProvider.AssertLocalServiceIsNotNull();
 
             this.errorListInfoBarController = this.extensionHost.GetService<IErrorListInfoBarController>();
             this.errorListInfoBarController.AssertLocalServiceIsNotNull();
+            this.errorListInfoBarController.Refresh();
 
-            this.CalculateSolutionBinding();
-        }
+            // The user changed the binding through the Team Explorer
+            this.extensionHost.VisualStateManager.BindingStateChanged += this.OnBindingStateChanged;
+
+            // The solution changed inside the IDE
+            this.solutionTracker.ActiveSolutionChanged += this.OnActiveSolutionChanged;
 
-        public bool IsActiveSolutionBound { get; private set; } = false;
+            this.isSolutionPreviouslyBound = this.IsActiveSolutionBound;
```

Comment:
Change IsActiveSolutionBound to 
IsActiveSolutionBound {get; private set} and there set its initial value from solutionBindingInformationProvider.IsSolutionBound